From c55874bc28f66d606527b940d4bd0f266bbaaa2c Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Mon, 17 Feb 2025 11:51:19 -0800
Subject: [PATCH] test: Including missing string.h for memcpy

Fixes
accept-test.c:39:2: error: call to undeclared library function 'memcpy' with type 'void *(void *, const void *, unsigned long)'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]

Upstream-Status: Submitted [https://github.com/axboe/liburing/pull/1351]
Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 test/232c93d07b74.c | 1 +
 test/accept-test.c  | 1 +
 2 files changed, 2 insertions(+)

diff --git a/test/232c93d07b74.c b/test/232c93d07b74.c
index d3053b9..d5677d3 100644
--- a/test/232c93d07b74.c
+++ b/test/232c93d07b74.c
@@ -10,6 +10,7 @@
 #include <stdlib.h>
 #include <stdint.h>
 #include <assert.h>
+#include <string.h>
 
 #include <pthread.h>
 #include <errno.h>
diff --git a/test/accept-test.c b/test/accept-test.c
index 99f6080..142b2de 100644
--- a/test/accept-test.c
+++ b/test/accept-test.c
@@ -7,6 +7,7 @@
 #include <sys/socket.h>
 #include <sys/un.h>
 #include <assert.h>
+#include <string.h>
 #include "liburing.h"
 #include "helpers.h"
 
